### ISSUE TITLE
Extend default compilation to other classifier models

### ISSUE THREAD

**Issue by jbischof** [keras-team/keras-nlp#709]
In #695 we added default compilation for `BertClassifier` and `RobertaClassifier` models. Extend this to the other classifier tasks currently in our library:
* [ ] `DistilBertClassifier`
* [ ] `DebertaV3Classifier`
* [ ] `AlbertClassifier`
* [ ] `FNetClassifier`
* [ ] `XLMRobertaClassifier`

For now, we will use `Adam` with a constant learning rate of `5e-5`:
```python
# Default compilation
self.compile(
    loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
    optimizer=keras.optimizers.Adam(5e-5),
    metrics=keras.metrics.SparseCategoricalAccuracy(),
    jit_compile=is_xla_compatible(self),
)
```

**Comment by Plutone11011** [comment]
Hi, may I work on this issue?

**Comment by jbischof** [comment]
Absolutely, thank you!
